DesignSafe-CI/portal
# task/DES-1829: DesignSafe Layout Redesign

## Overview: ##
This PR includes a site redesign, focusing on the header, footer, and body layout.

Design provided by CMD: https://xd.adobe.com/view/cca67126-a94d-401d-42dd-44c7e8ff1f76-ba97/

## PR Status: ##


## Related Jira tickets: ##

* [DES-1829](https://jira.tacc.utexas.edu/browse/DES-1829)

## Summary of Changes: ##
- New, slimmer DesignSafe logo
- Second header at the top, with NSF and NHERI branding
- New footer layout
- Body layout utilizes a css grid layout
- Navbar redesign: color and no longer "sticky" (yay!)
- "Research Workbench" renamed to "Workspace"
- "Workspace" renamed to "Tools and Applications"

## Testing Instructions:
1. Run the portal locally
2. Ensure layout is consistent with design, and fairly responsive on multiple views (large screen, small screen, etc)

## UI Photos:
Header and Body:
<img width="1791" alt="Screen Shot 2020-12-11 at 7 33 08 PM" src="https://user-images.githubusercontent.com/20326896/101969026-cb58a300-3be7-11eb-82b0-0f5a3188f636.png">


Footer:
<img width="1788" alt="Screen Shot 2020-12-11 at 7 33 23 PM" src="https://user-images.githubusercontent.com/20326896/101969027-cbf13980-3be7-11eb-877f-aafb7aa3e8c5.png">
